node/src/Pos/Launcher/Runner.hs

```diff
@@ -121,7 +125,19 @@ runRealModeDo NodeResources {..} outSpecs action =
     NetworkConfig {..} = ncNetworkConfig
     NodeParams {..} = ncNodeParams
     LoggingParams {..} = bpLoggingParams npBaseParams
-    startMonitoring oq node' =
+
+    -- Expose the health-check endpoint for DNS load-balancing
+    -- and optionally other services as EKG & statsd.
+    startMonitoring topology oq node' = do
+        -- Expose the health-check
+        let listeningPort = case ncTcpAddr of
+                TCP.Unaddressable                             -> ncDefaultPort
+                TCP.Addressable (TCP.tcpBindPort -> bindPort) -> fromMaybe ncDefaultPort (readMaybe bindPort)
+        mRoute53HealthCheck <- case topologyRoute53HealthCheckEnabled topology of
+            False -> return Nothing
+            True  -> let app = route53HealthCheckApplication topology oq
+                     in Just <$> async (serveImpl app "127.0.0.1" (listeningPort + 30) Nothing)
```

Comment:
This is the only thing that bothers me. Too much trouble to make the port configurable? I take it there's nothing special about `30` right?

Reply:
> This is the only thing that bothers me. Too much trouble to make the port configurable?

Yes, I'm not particularly proud of this piece of code either, but adding an extra parameter for something we do want to be the same for each node I thought was more trouble than gain. I would be curious to hear Devops' perspective, as I guess we should go with whatever makes their lives easier, and I will cope 😉 

> I take it there's nothing special about 30 right?

Just a random constant. Originally I tried to bind to `bindPort + 1` but that was a no-go as it was screwing up the topology files we had for the local benchmarks.

Reply:
@adinapoli-iohk I think we need to make this port configurable. I don't know what port it will be and it may have to be a specific one. We should let devops decide the port by making it configurable.